AreaOfTriangle.rb

```diff
@@ -0,0 +1,9 @@
+class Triangle
+  VALUE = 0.5
```

Comment:
constant should define in the top of file, under class name

Reply:
in this case, is it not under the class name?